### PR TITLE
Styiling feature

### PR DESCRIPTION
**Pull request by Mar12358**
### Added features:
- Add style

See more info [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md).

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -2,7 +2,6 @@ import { Routes, Route } from 'react-router-dom';
 import Home from './routes/Home';
 import Books from './routes/Books';
 // import { useState } from 'react'
```
Comment (AnselemOdims) [inline src/App.jsx]: - [ ] Please remove any form of commented code in your codebase

File: src/index.css [hunk 2]
```diff
@@ -1,102 +1,154 @@
...
-    background-color: #f9f9f9;
-  }
+.link-btn {
+  background: none;
+  border: none;
+  color: rgb(129, 129, 253);
 }
 
-nav > ul {
+.buttons {
```
Comment (AnselemOdims) [inline src/index.css]: - [ ] Good job working on the styling but there are just a few things that you need to address. Please pay great attention to the sections highlighted.
- [ ] Please kindly ensure that your design matches the original design guidelines(font size, background-color, color, padding, margin, etc)

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Mar12358 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by fobadara — APPROVED** [review]
Hi @Mar12358, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Implemented reviewer's suggestions

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**